edge-hub/test/Microsoft.Azure.Devices.Routing.Core.Test/endpoints/StoringAsyncEndpointExecutorTest.cs

```diff
@@ -25,23 +26,26 @@ public async Task InvokeTest()
             // Arrange
             const int MessagesCount = 10;
             const string EndpointId = "endpoint1";
+            const uint Priority = 100;
             var endpoint = new NullEndpoint(EndpointId);
-            var priorities = new List<uint>() { 0, 1, 2, 100, 101, 102 };
-            var checkpointer = new NullCheckpointer();
+            var priorities = new List<uint>() { Priority };
```

Comment:
why this reduce priority test data to 1 only?

Reply:
you simplify this test here for testing basic scenario and have the new test "MessagePrioritiesTest" for different priority. right?

Reply:
correct, I previously had multiple priorities here because I needed a way to locally verify my last change.  Now that the feature is done E2E, I was able to write a new, more comprehensive testcase, so this isn't needed here anymore.